Dockerfile

```diff
@@ -100,7 +114,8 @@ ARG CGO_ENABLED=0
 RUN --mount=target=. --mount=target=/root/.cache,type=cache \
   --mount=target=/go/pkg/mod,type=cache \
   --mount=source=/tmp/.ldflags,target=/tmp/.ldflags,from=buildkit-version \
-  xx-go build ${GOBUILDFLAGS} -ldflags "$(cat /tmp/.ldflags) -extldflags '-static'" -tags "osusergo netgo static_build seccomp ${BUILDKITD_TAGS}" -o /usr/bin/buildkitd ./cmd/buildkitd && \
+  --mount=source=/tmp/.gcflags,target=/tmp/.gcflags,from=buildkit-version \
```

Comment:
Better to put the files in same subdir in the version stage and mount that dir instead of mounting separate files.

Reply:
Reverting this as mentioned above.